metricbeat/module/docker/image/image.go

```diff
@@ -14,9 +14,9 @@ import (
 // init registers the MetricSet with the central registry.
 // The New method will be called after the setup of the module and before starting to fetch data
 func init() {
-	if err := mb.Registry.AddMetricSet("docker", "image", New); err != nil {
-		panic(err)
-	}
+	mb.Registry.MustAddMetricSet("docker", "image", New,
+		mb.WithHostParser(docker.HostParser),
```

Comment:
It had no host parser before :thinking: but I guess it should have had it.

Reply:
Hm, good point. Yes it should have one.